pkg/action/action.go

```diff
@@ -91,6 +93,76 @@ type Configuration struct {
 	mutex sync.Mutex
 }
 
+const (
+	// FilenameAnnotation is the annotation key used to store the original filename
+	// information in manifest annotations for post-rendering reconstruction.
+	FilenameAnnotation = "helm-postrender-filename"
```

Comment:
Can we make this private? Unless there is a good reason to make it public, and so part of Helm's SDK public interface, private is preferred. As that allows it to remain an implementation detail, and able to be changed in the future. If public, it becomes part of Helm's public SDK interface, and non-amendable to changes/non-backwards compatible updates in the future.

Reply:
Done